src/api/test/test_helper.rb

```diff
@@ -138,7 +140,7 @@ class ActionDispatch::IntegrationTest
   # but for debugging or for deep testing the check can be run after each test case
   def after_teardown
     super
-    if false
+    if DEBUG_MODE
```

Comment:
@hennevogel probably ... What do you prefer? Adding the variable or commenting the whole block?